index.html

```diff
@@ -1,225 +1,320 @@
-                </div>
-                <div class="work_details">
-                    <h2 class="work_title">Tonic</h2>
-                    <ul class="work_info">
-                        <li class="w_info_item">CANOPY</li>
-                        <li class="w_info_item">Back End Dev</li>
-                        <li class="w_info_item">2015</li>
-                    </ul>
-                    <p class="work_details_content">
-                        A daily selection of privately personalized reads; no accounts or sign-ups required.
-                    </p>
-                    <ul class="work_cats">
-                        <li>Html</li>
-                        <li>Css</li>
-                        <li>Javascript</li>
-                    </ul>
-                    <button type="button" class="default_btn">See More</button>
-                </div>
-            </div>
-
-            <!-- work card 2 -->
-            <div class="work_card">
-                <div class="work_preview">
-                    <img src="./images/Snapshoot Portfolio (1).png" alt="Project 2" />
-                </div>
-                <div class="work_details">
-                    <h2 class="work_title">Multi-Post Stories</h2>
-                    <ul class="work_info">
-                        <li class="w_info_item">CANOPY</li>
-                        <li class="w_info_item">Back End Dev</li>
-                        <li class="w_info_item">2015</li>
-                    </ul>
-                    <p class="work_details_content">
-                        A daily selection of privately personalized reads; no accounts or sign-ups required.
-                    </p>
-                    <ul class="work_cats">
-                        <li>Html</li>
-                        <li>Css</li>
-                        <li>Javascript</li>
-                    </ul>
-                    <button type="button" class="default_btn">See More</button>
-                </div>
-            </div>
+      <!-- Work section -->
+      <section class="work">
+        <!-- work card 1 -->
+        <div class="work_card">
+          <div class="work_preview">
+            <img src="./images/Snapshoot Portfolio.png" alt="Project 1" />
+          </div>
+          <div class="work_details">
+            <h2 class="work_title">Tonic</h2>
+            <ul class="work_info">
+              <li class="w_info_item">CANOPY</li>
+              <li class="w_info_item">Back End Dev</li>
+              <li class="w_info_item">2015</li>
+            </ul>
+            <p class="work_details_content">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="work_cats">
+              <li>Html</li>
+              <li>Css</li>
+              <li>Javascript</li>
+            </ul>
+            <button type="button" class="default_btn">See More</button>
+          </div>
+        </div>
```

Comment:
- [ ] Looking at the design on the Figma template. The card is divided into two equal parts. One for the image and the other one for the image info. In your own case, the info part takes a larger part of the card which is not in line with what we have on the guiding template. Kindly make your project to produce a close replica of what we have on the guiding template. This applies to all other work-cards in your project. Take a look at the both designs and work your design along side with the guiding template.

### Your Design

![your-project-card](https://user-images.githubusercontent.com/26597432/200880047-ae887876-af15-45a7-a0aa-1cdae9843937.png)

### Figma Design

![figma-project-card](https://user-images.githubusercontent.com/26597432/200880355-56bcad49-19b1-4f77-b827-6cf9a496f664.png)